### PR TITLE
npm audit: add page 

### PR DESCRIPTION
**Pull request by bharateshwq**
#14000

- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page(s) follow the [style guide](/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message-and-pr-title).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages/common/npm-audit.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+# npm audit
+
+> Command to scan for known vulnerabilities in project dependencies.
```
Comment (spageektti) [inline pages/common/npm-audit.md:3]:
```suggestion
> Scan for known vulnerabilities in project dependencies.
```

File: pages/common/npm-audit.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+- Perform a dry run-  Simulates the fix process without making any changes:
+
+`npm audit fix --dry-run`
+
+- Output audit results in JSON format-  Provides audit results in a structured JSON format for processing:
+
+`npm audit --json`
+
+- Set a minimum severity level- Configures the audit to only fail on vulnerabilities above a specified severity:
```
Comment (spageektti) [inline pages/common/npm-audit.md:27]:
```suggestion
- Scan project’s dependencies for known vulnerabilities:

`npm audit`

- Automatically fix vulnerabilities:

`npm audit fix`

- Update the lock file without modifying the `node_modules` directory:

`npm audit fix --package-lock-only`

- Perform a dry run. Simulate the fix process without making any changes:

`npm audit fix --dry-run`

- Output audit results in JSON format:

`npm audit --json`

- Configure the audit to only fail on vulnerabilities above a specified severity:
```

Comment (bharateshwq) [reply pages/common/npm-audit.md:27]: got it this will help me push better docs
thanks! @spageektti

File: pages/common/npm-audit.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
+# npm audit
+
+> Scan for known vulnerabilities in project dependencies.
+> Reports vulnerabilities and suggests remediation.
+> More information: <https://docs.npmjs.com/cli/npm-audit>.
+
+- Scan project’s dependencies for known vulnerabilities:
```
Comment (kbdharun) [inline pages/common/npm-audit.md:7]:
```suggestion
- Scan the project’s dependencies for known vulnerabilities:
```

File: pages/common/npm-audit.md [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+
+`npm audit`
+
+- Automatically fix vulnerabilities:
+
+`npm audit fix`
+
```
Comment (kbdharun) [inline pages/common/npm-audit.md:14]:
```suggestion

- Force an automatic fix to dependencies with vulnerabilities:

`npm audit fix {{-f|--force}}`

```

File: pages/common/npm-audit.md [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+> Scan for known vulnerabilities in project dependencies.
+> Reports vulnerabilities and suggests remediation.
+> More information: <https://docs.npmjs.com/cli/npm-audit>.
+
+- Scan project’s dependencies for known vulnerabilities:
+
+`npm audit`
+
+- Automatically fix vulnerabilities:
```
Comment (kbdharun) [inline pages/common/npm-audit.md:11]:
```suggestion
- Automatically fix vulnerabilities in the project's dependencies:
```

**Review by kbdharun — COMMENTED** [review]
Thanks for your contribution. I have some minor suggestions for this page.

**Review by kbdharun — APPROVED** [review]

**Review by spageektti — APPROVED** [review]
Thanks

**Pull request merged**